src/main/java/com/jc/tm/database/dao/CommentDaoImpl.java

```diff
@@ -0,0 +1,144 @@
+    private static final String GET_BY_ID = "SELECT ID, TASK_ID, TEXT, CREATED FROM COMMENT WHERE ID = ? ";
+    private static final String GET_ALL = "SELECT ID, TASK_ID, TEXT, CREATED FROM COMMENT";
+    private static final String DELETE = "DELETE FROM COMMENT WHERE ID = ? ";
+
+    Connection connection = Jdbc.getConnection();
+
+    @Override
+    public void insert(Comment comment) throws SQLException {
+        PreparedStatement preparedStatement = null;
+        try {
+            preparedStatement = connection.prepareStatement(INSERT);
+            preparedStatement.setLong(1, comment.getTaskId());
+            preparedStatement.setString(2, comment.getText());
+            preparedStatement.setTimestamp(3, java.sql.Timestamp.valueOf(java.time.LocalDateTime.now()));
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            //gonna log this later
+            System.out.println(e.getMessage());
+            connection.rollback();
+        } finally {
+            if (preparedStatement != null) {
+                preparedStatement.close();
+            }
+            if (connection != null) {
+                connection.close();
+            }
+        }
+    }
+
+    @Override
+    public void update(Comment comment) throws SQLException {
+        PreparedStatement preparedStatement = null;
+        try {
+            preparedStatement = connection.prepareStatement(UPDATE);
+            preparedStatement.setString(1, comment.getText());
+            preparedStatement.setTimestamp(2, java.sql.Timestamp.valueOf(java.time.LocalDateTime.now()));
+            preparedStatement.setLong(3, comment.getId());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            //gonna log this later
+            System.out.println(e.getMessage());
+        } finally {
+            if (preparedStatement != null) {
+                preparedStatement.close();
+            }
+            if (connection != null) {
+                connection.close();
+            }
+        }
+    }
+
+    @Override
+    public Comment getById(Long id) throws SQLException {
+        PreparedStatement preparedStatement = null;
+        Comment comment = new Comment();
+        try {
+            preparedStatement = connection.prepareStatement(GET_BY_ID);
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while(resultSet.next()) {
+                comment.setId(resultSet.getLong(ID));
+                comment.setTaskId(resultSet.getLong(TASK_ID));
+                comment.setText(resultSet.getString(TEXT));
+                comment.setCreated(resultSet.getTimestamp(CREATED).toLocalDateTime());
+            }
+            preparedStatement.execute();
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
```

Comment:
1. do not user system.out
2. if we will get error wheen call database, what result will be of this method?